sci-libs/libgeodecomp/libgeodecomp-0.4.0-r1.ebuild

```diff
@@ -31,7 +31,11 @@ DEPEND="${RDEPEND}
 	visit? ( sci-visualization/visit )"
```

Comment:
`sci-libs/libgeodecomp[visit]` had to be masked due to the dependency having been last-rited: https://github.com/gentoo/gentoo/blob/master/profiles/base/package.use.mask#L63

This PR is an excellent opportunity to remove the option from the ebuild.